docs/plugins.md

```diff
@@ -14,183 +14,157 @@ Looking for help using Snowpack in your project?
 
 ## Overview
 
-Snowpack runs through 3 stages internally: _build_, _transform_, and _bundle_. Snowpack plugins hook into **one** of these stages:
+There are two types of Snowpack plugins: **build plugins** and **bundler plugins.** Unless you are making a plugin specifically for a bundler like Rollup, Parcel, or webpack, you’re probably making a build plugin.
 
-<ol class="ol">
-  <li class="tc-green">
-    <h4 class="ol-title">Build</h4>
-    <div class="tc-text">The main action Snowpack uses. This handles most operations, such as transforming JSX to JS, or converting Sass to CSS. Every build plugin should map to an existing <a href="/#build-scripts">build script</a>.</div>
-  </li>
-  <li class="tc-blue">
-    <h4 class="ol-title">Transform</h4>
-    <div class="tc-text">This stage happens after the <strong>build</strong> step, and can be used for additional transformations if needed. This has more context since it comes later, such as which URL the module was requested at.</div>
-  </li>
-  <li class="tc-magenta">
-    <h4 class="ol-title">Bundle</h4>
-    <div class="tc-text">Optional stage that only happens when using a bundler plugin such as <a href="https://github.com/pikapkg/create-snowpack-app/tree/master/packages/plugin-webpack" target="_blank" rel="noopener nofollow">@snowpack/plugin-webpack</a> or <a href="https://github.com/pikapkg/create-snowpack-app/tree/master/packages/plugin-parcel" target="_blank" rel="noopener nofollow">@snowpack/plugin-parcel</a>. This passes all of Snowpack’s built files to a final bundler. <em>Note: only 1 bundler plugin can receive the final files.</em></div>
-  </li>
-</ol>
+## Build Plugins
 
+Build plugins run on single files that pass through Snowpack. They listen for particular file extensions, and when a match is found, transform that code.
 
-## Plugin API
+In order to keep operations speedy, Snowpack’s build system is **single pass,** meaning each file is only transformed by a set of plugins once. We’ll talk more about that later.
 
-A Snowpack plugin is a JavaScript module that exports a function. That function takes Snowpack config & plugin options as it's function arguments and must return a valid Snowpack Plugin object.
-
-### Example
+### The Basics
 
 ```js
-// snowpack.config.json
-"plugins": [["@prefresh/snowpack", { /* `pluginOptions` (optional) */ }]]
+// Snowpack plugin template
+
+module.exports = (snowpackConfig, pluginOptions) => ({
```

Comment:
Love how concise we were able to get this, but I worry that the terse syntax hides the fact that this is a function. I always try to go for verbose / avoid new syntax in docs in case users aren't familiar with the new hotness, something like:

```js
module.exports = function(snowpackConfig, pluginOptions) {
  // Optional: do any set up here
  return {
    // ..
  };
}
```

Reply:
Good callout. Updated.